manifests/init.pp

```diff
@@ -88,6 +88,7 @@
 # @param cvmfs_repo_list Specify exactly the REPO_LIST in `defaults.local` overriding auto population.
 # @param cvmfs_alien_cache Use an alien cache
 # @param cvmfs_shared_cache Enable a shared cache
+# @param cvmfs_include_fsck Ensure the cvmfs::fsck class is included.
```

Comment:
```suggestion
# @param cvmfs_fsck Ensure the cvmfs::fsck class is included.
```
Don't think we want include in variable name as the method does not need to be exposed.